### PR TITLE
fix: fixing scroll, z-index and removing a dash

### PR DESCRIPTION
**Pull request by aballerr**
Tasks
https://uniswaplabs.atlassian.net/browse/WEB-985
https://uniswaplabs.atlassian.net/browse/WEB-953
z-index: I noticed this when using the site that there is overlap of the navbar and widget
Edit: z-index addressed here https://github.com/Uniswap/interface/pull/4576

Screen Shot 2022-08-31 at 12 42 43 PM
Screen Shot 2022-08-31 at 1 11 42 PM
Screen Shot 2022-08-31 at 1 11 49 PM

### CODE REVIEW

File: src/components/Dialog.tsx [hunk 1]
```diff
@@ -111,7 +111,7 @@ export const Modal = styled.div<{ color: Color }>`
   position: absolute;
   top: 0;
   width: 100%;
-  z-index: ${Layer.DIALOG};
+  z-index: 1;
```
Comment (aballerr) [inline src/components/Dialog.tsx:114]: there isn't a layer for 1, I can add one if wanted

File: src/components/Swap/Settings/index.tsx [hunk 2]
```diff
@@ -33,7 +31,7 @@ export function SettingsDialog() {
           </ThemedText.ButtonSmall>
         </TextButton>
       </Header>
-      <Column gap={1} style={{ paddingTop: '1em' }} ref={setBoundary} padded css={scrollbar}>
+      <Column gap={1} style={{ paddingTop: '1em' }} ref={setBoundary} padded>
```
Comment (aballerr) [inline src/components/Swap/Settings/index.tsx:34]: It's not obvious to me that we need this scrolling styling.  I believe we are okay to remove it

File: src/components/Dialog.tsx (hunk 1 above)
Comment (zzmp) [reply src/components/Dialog.tsx:114]: Why are you not using `Layer.DIALOG` for the `Dialog`?
If this is shared by another layer, then _that_ layer should be renamed.

File: src/components/Swap/Settings/index.tsx (hunk 2 above)
Comment (zzmp) [reply src/components/Swap/Settings/index.tsx:34]: It used to be necessary (there were more settings), but with the pared down settings menu there should never be a scrollbar 👍

**Review by zzmp — CHANGES_REQUESTED** [review]
This is blocked by `Layer`. If it's easy to make it three separate PRs it'll get the other two bits in faster.

File: src/components/Dialog.tsx (hunk 1 above)
Comment (aballerr) [reply src/components/Dialog.tsx:114]: if you look at the description you can see the modal is overlapping on the navbar.  Changing this to a z-index of 1 fixes that.

Comment (zzmp) [reply src/components/Dialog.tsx:114]: z-index's should only be done through the `Layer` enum to keep them organized.
This should be updated in src/theme/layer.ts, and continue using `Layer.DIALOG`.

**Review by zzmp — CHANGES_REQUESTED** [review]

File: src/components/Dialog.tsx (hunk 1 above)
Comment (aballerr) [reply src/components/Dialog.tsx:114]: lynn solved this issue here: https://github.com/Uniswap/interface/pull/4576.  I'm going to revert

**Review by zzmp — APPROVED** [review]

**Pull request merged**